### ISSUE TITLE
示例代码中 self.crf_loss = LinearChainCrfLoss(self.crf) 总是报 `missing 1 required positional argument: 'labels'`

### ISSUE THREAD

**Issue by jiaqianjing** [PaddlePaddle/PaddleNLP#452]
https://github.com/PaddlePaddle/PaddleNLP/blob/develop/examples/information_extraction/waybill_ie/run_ernie_crf.py
请确认一下这个 example 是否有问题。每次在计算前项 loss 的时候，总是报如题所示的问题，我检查过数据，都没有问题，另外，最后我还用你们仓库下的数据跑，依然出现这个问题。

**Comment by jiaqianjing** [comment]
应该不止用 ernie + CRF 有问题，只要网络后面套了 CRF 的都有问题

**Comment by joey12300** [comment]
@jiaqianjing 您好，请问您使用的PaddleNLP的版本是多少？能提供更详细的报错信息吗？

**Comment by jiaqianjing** [comment]
pip list 出来的版本是：paddlenlp==2.0.0 , 用的就是 aistudio .
不过我好像找到问题了：
![image](https://user-images.githubusercontent.com/16071449/119760797-41726d00-bedd-11eb-8e46-04cf62023bab.png)
![image](https://user-images.githubusercontent.com/16071449/119760824-50f1b600-bedd-11eb-8d8a-7b4a4ac6ab03.png)
我看你们 github 上的代码不一样，这个 `LinearChainCrfLoss ` 多了一个参数，不知道这是哪个版本的。。。

**Comment by joey12300** [comment]
请先卸载您当前版本的paddlenlp：`pip uninstall paddlenlp`。重新安装paddlenlp：`pip install --upgrade paddlenlp -i https://pypi.org/simple`。并更新教程实例，重新跑一遍。

**Comment by jiaqianjing** [comment]
> 请先卸载您当前版本的paddlenlp：`pip uninstall paddlenlp`。重新安装paddlenlp：`pip install --upgrade paddlenlp -i https://pypi.org/simple`。并更新教程实例，重新跑一遍。
嗯，现在看起来版本对的上了，不过出现了一个新的错误。
![image](https://user-images.githubusercontent.com/16071449/119763718-a2507400-bee2-11eb-8fc3-3ee98bd481cc.png)
我的训练数据样本不多，就 162 条，我batch_size=128, 这个有什么参数可以 drop 不掉不够 batch_size 数据的参数吗

**Comment by jiaqianjing** [comment]
> 请先卸载您当前版本的paddlenlp：`pip uninstall paddlenlp`。重新安装paddlenlp：`pip install --upgrade paddlenlp -i https://pypi.org/simple`。并更新教程实例，重新跑一遍。

已经可以了，3Q.
